e2e-tests/commands/insurance/check-export-contract-summary-list.js

```diff
@@ -217,6 +217,19 @@ const checkExportContractSummaryList = {
       cy.assertSummaryListRowDoesNotExist(summaryList, fieldId);
     }
   },
+  [FIXED_SUM_CURRENCY_CODE]: ({ shouldRender = false }) => {
+    const fieldId = FIXED_SUM_CURRENCY_CODE;
+
+    if (shouldRender) {
+      const { expectedKey, expectedChangeLinkText } = getSummaryListField(fieldId, FIELDS.AGENT_CHARGES);
+
+      const expectedValue = application.EXPORT_CONTRACT.AGENT_CHARGES[fieldId];
```

Comment:
could we dereference EXPORT_CONTRACT at the top?

Reply:
We could, but I think it would become less clear. Having `application` makes it very clear, especially when there are other tests in the same file that have expected values that do not come from the `application` object.